### ISSUE TITLE
Missing style.css

### ISSUE THREAD

**Issue by philipempl** [dash14/v-network-graph#3]
Hi,
First, I appreciate your library much, and i would like to thank you for your effort. I sadly encountered that the import of the style.css file is failing. It seems to be that the styles.css file (import "v-network-graph/style.css) is missing in the node_modules directory. Do you have any workaround for that?
Thanks in advance,
Have a nice day. 
Phil

**Comment by TBSI-Seth** [comment]
I had this same issue.  There are two ways to fix it that I found and dash14 did respond to my issue with a solution as well. So I'll pass it on. The style.css was in the lib folder so all I had to do to make it work for me was change the import statement to point to **v-network-graph/lib/style.css**

This was the solution proposed by dash14 and I assume this would work equally well.
```
In the package.json, I have specified the following mapping so that it can be loaded with import "v-network-graph/style.css".
  "exports": {
    ...
    ". /style.css": ". /lib/style.css"
  },
```

**Comment by dash14** [comment]
Hi @philipempl,
Thank you for your report.

And, @TBSI-Seth, thank you for your kind cooperation.

I think some people will have the same problem in the future, so I decided to change it so that the following paths can be imported regardless of whether the environment supports "subpath exports" of package.json or not.

```js
import "v-network-graph/lib/style.css"
```

I just released v0.1.6, which supports the above path.
